clang/include/clang/AST/PrettyPrinter.h

```diff
@@ -121,6 +123,7 @@ struct PrintingPolicy {
   /// \endcode
   LLVM_PREFERRED_TYPE(bool)
   unsigned SuppressTagKeyword : 1;
+  unsigned SuppressTagKeywordInAnonymousTagNames : 1;
```

Comment:
Maybe better to turn this into an 3 valued enum.

Or at least document this separatelly.

Reply:
Enum sounds like a good compromise, let me try and see what that would look like

Reply:
Made it an enum in one of the latest commits. Bit awkward with the casting, but nothing we can do about it if the bitfields are `unsigned` i guess

I'll extract it into a separate PR to make it easier to review